.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+###################################
+##### Global Protection Rule ######
+###################################
+# NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
+*                                               @hashgraph/hedera-sdk @hashgraph/hedera-sdk-cpp-maintainers
```

Comment:
Not sure why Github is saying `@hashgraph/hedera-sdk-cpp-maintainers` is an unknown owner, is this something to be concerned with?

Reply:
I forgot to assign the team to this repo. It's now showing as "valid"